run-server.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+sudo /proj/demeter-PG0/prthaker/cornflakes/mlx5-netperf/build/mlx5-netperf \
+     --mode=SERVER \
+     --pci_addr=0000:41:00.0 \
+     --server_mac=$SERVER_MAC \
+     --server_ip=$SERVER_STATIC_IP \
+     --client_mac=$CLIENT_MAC \
+     --client_ip=$CLIENT_STATIC_IP \
+     --array_size=32768000 \
+     --segment_size=512 \
+     --num_segments=2 \
+     --num_cores=2
+#     --with_copy
```

Comment:
Could you remove these files or alteast make the folder path and pci address variables?